### ISSUE TITLE
How do I make it so it auto selects to play an another game?

### ISSUE THREAD

**Issue by TheUknown23** [coryshaw1/saliens-hack#4]

**Comment by coryshaw1** [comment]
Part of the script checks for the round ending message, and then should continue and select a new zone automatically. Do you see any errors in console when this bug occurs?

**Comment by TheUknown23** [comment]
so I think you have to let the script press the play button by itself. If you press play then i don't think it picks the planets and stuff. I may be wrong though.

**Comment by coryshaw1** [comment]
It should press the play button, I haven't added support to select planets yet. You have to select the planet, and then it will autoselect the zones within the planet.

**Comment by coryshaw1** [comment]
Planet auto-selection has been added. Are you still seeing this bug where a new round doesn't start?

**Comment by TheUknown23** [comment]
No, I don't see the bug but it looks the program will not work when the
user presses the play button. You have to let the script press the button
to start itself. This is not gamebreaking and I actually like it.
